src/goto-symex/symex_function_call.cpp

```diff
@@ -53,8 +53,7 @@ void goto_symext::parameter_assignments(
 
     const irep_idt &identifier=parameter.get_identifier();
 
-    if(identifier.empty())
-      throw "no identifier for function parameter";
+    INVARIANT(!identifier.empty(), "no identifier for function parameter");
```

Comment:
Again, I'd switch the tone around to positive.

Reply:
Agree with Hannes - something like "function pointer parameter must have an identifier"